### PR TITLE
Localize placeholder based on first language

### PR DESCRIPTION
**Pull request by scottsfarley93**
Implements the change suggested in #150 by localizing the placeholder value based on the provided language option set in the constructor.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -388,6 +389,24 @@ MapboxGeocoder.prototype = {
...
   },
 
+  /**
+   * Get the text to use as the search bar placeholder
+   * 
+   * If placeholder is provided in options, then use options.placeholder
+   * Otherwise, if language is provided in options, then use the localized string of the first language if available
+   * Otherwise use the default
+   */
+  getPlaceholderText: function(){
```
Comment (andrewharvey) [inline lib/index.js]: should this method be private?

File: lib/index.js [hunk 2]
```diff
@@ -388,6 +389,24 @@ MapboxGeocoder.prototype = {
...
+    if (this.options.placeholder) return this.options.placeholder;
+    if (this.options.language){
+      var firstLanguage = this.options.language.split(",")[0];
+      var language = subtag.language(firstLanguage);
+      var localizedValue = localization.placeholder[language];
+      if (localizedValue)  return localizedValue;
+    }
+    return 'Search';
```
Comment (andrewharvey) [inline lib/index.js]: you could also lookup 'en' in localization.placeholder, but I don't think it really matters.

File: lib/index.js [hunk 3]
```diff
@@ -388,6 +389,24 @@ MapboxGeocoder.prototype = {
     return this.options.proximity;
   },
 
+  /**
+   * Get the text to use as the search bar placeholder
+   * 
+   * If placeholder is provided in options, then use options.placeholder
+   * Otherwise, if language is provided in options, then use the localized string of the first language if available
+   * Otherwise use the default
```
Comment (andrewharvey) [inline lib/index.js]: I think we should add a JSDoc `@returns` here for completeness.

File: lib/index.js (hunk 1 above)
Comment (andrewharvey) [reply lib/index.js]: related to https://github.com/mapbox/mapbox-gl-geocoder/pull/198/files#r265396617 but I think it's better to re-gen the docs with each commit and include them.

Comment (scottsfarley93) [reply lib/index.js]: Good catch. Moved this to be a private method.

**Pull request merged**